edgraph/access_ee.go

```diff
@@ -351,7 +351,13 @@ func RefreshAcls(closer *z.Closer) {
 		if kvs == nil || len(kvs.Kv) == 0 {
 			return
 		}
```

Comment:
Is it even possible that we get an empty list? If yes, then we should have this check at other places as well.